common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, print_function
+
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the last six months and
```

Comment:
I am confused with the docstring and the actual timeline of the users being retrieved(six months vs 1 week). Is one week being used for testing?